src/asmcnc/comms/grbl_settings_manager.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, machine=None):
             self.machine.s.bind(setting_121=lambda instance, value: self.on_persistent_setting(value, 121))
             self.machine.s.bind(setting_122=lambda instance, value: self.on_persistent_setting(value, 122))
             self.machine.s.bind(setting_130=lambda instance, value: self.on_persistent_setting(value, 130))
```

Comment:
Partial would be nicer than lambda. I mentioned this to Vlad and Benji in separate PRs, with a lambda, you can't unbind (if you needed to) and I think event functions should follow the intended signature for consistency and readability:
`def x(instance, value)`

Reply:
Happy to merge as this isn't directly related to this ticket.

Reply:
Well, I didn't like that the function then has a parameter that is never used, so I went with the lambda. Sure partial would work as well. Maybe we'll rework it at some point...